src/main/java/decodex/data/exception/RecipeException.java

```diff
@@ -0,0 +1,13 @@
+package decodex.data.exception;
+
+/**
+ * This exception is raised when an error during recipe execution occurs.
+ */
+public class RecipeException extends Exception {
+
+    public static final String EMPTY_RECIPE_MESSAGE = "[x] There is nothing to be removed.";
```

Comment:
Perhaps the message can be "The recipe is currently empty. There is no module to be removed." to be more descriptive.